gc/base/MemoryManager.cpp

```diff
@@ -244,8 +250,35 @@ MM_MemoryManager::createVirtualMemoryForHeap(MM_EnvironmentBase *env, MM_MemoryH
 
 						if ((NULL == instance) && (ceiling > (void *)NON_SCALING_LOW_MEMORY_HEAP_CEILING)) {
 
+#if defined(S390)
```

Comment:
I see a few spots that use more specific check for zLinux

defined(LINUX) && defined(S390)